build.sbt

```diff
@@ -0,0 +1,81 @@
+/**
+ * Copyright (c) 2014-2017 Snowplow Analytics Ltd. All rights reserved.
+ *
+ * This program is licensed to you under the Apache License Version 2.0,
+ * and you may not use this file except in compliance with the Apache License Version 2.0.
+ * You may obtain a copy of the Apache License Version 2.0 at http://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the Apache License Version 2.0 is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the Apache License Version 2.0 for the specific language governing permissions and limitations there under.
+ */
+
+lazy val commonDependencies = Seq(
+  // Java
+  Dependencies.Libraries.config,
+  Dependencies.Libraries.slf4j,
+  Dependencies.Libraries.log4jOverSlf4j,
+  Dependencies.Libraries.kinesisClient,
+  Dependencies.Libraries.kinesisConnector,
+  Dependencies.Libraries.validator,
+  // Scala
+  Dependencies.Libraries.scopt,
+  Dependencies.Libraries.scalaz7,
+  Dependencies.Libraries.scalazC7,
+  Dependencies.Libraries.snowplowTracker,
+  Dependencies.Libraries.analyticsSDK,
+  Dependencies.Libraries.awsSigner,
+  // Scala (test only)
+  Dependencies.Libraries.specs2
+)
+
+lazy val buildSettings = Seq(
+  organization  := "com.snowplowanalytics",
+  name          := "snowplow-elasticsearch-loader",
+  version       := "0.9.0-rc4",
+  description   := "Load the contents of a Kinesis stream to Elasticsearch",
+  scalaVersion  := "2.11.11",
+  scalacOptions := BuildSettings.compilerOptions,
+  javacOptions  := BuildSettings.javaCompilerOptions,
+  resolvers     += Resolver.jcenterRepo,
+  shellPrompt   := { _ => "elasticsearch-loader> " }
```

Comment:
I still think that  `moduleName.value + "> "` is better than `"elasticsearch-loader> "` as you have an idea on what SBT subproject you're working on.